array_api_strict/_searching_functions.py

```diff
@@ -80,6 +80,9 @@ def where(condition: Array, x1: Array, x2: Array, /) -> Array:
     """
     # Call result type here just to raise on disallowed type combinations
     _result_type(x1.dtype, x2.dtype)
+    
+    if condition.dtype != bool:
```

Comment:
Should be `array_api_strict.bool != builtins.bool`

Reply:
Ah good catch, I missed the override of the built-in